rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -3418,4 +3420,56 @@ public ResponseEntity<?> createDuplicateProjectWithDependencyNetwork(
 
         return ResponseEntity.created(location).body(projectDTOHalResource);
     }
+
+    @Operation(
+            summary = "Add licenses to linked releases of a project.",
+            description = "This API adds license information to linked releases of a project by processing the approved CLI attachments for each release. It categorizes releases based on the number of CLI attachments (single, multiple, or none) and updates their main and other licenses accordingly.",
+            tags = {"Project"},
+                    parameters = {
+                            @Parameter(
+                                name = "projectId",
+                                description = "The ID of the project whose linked releases need license updates.",
+                                required = true,
+                                example = "12345",
+                                schema = @Schema(type = "string")
+                            )
+                        },
+            responses = {
+                    @ApiResponse(
+                            responseCode = "200",
+                            description = "License information successfully added to linked releases.",
+                            content = @Content(
+                                mediaType = "application/hal+json",
+                                schema = @Schema(type = "object", implementation = JSONObject.class),
+                                examples = @ExampleObject(
+                                    value = "{\n  \"one\": [\"Release1\", \"Release2\"],\n  \"mul\": [\"Release3\"]\n}"
```

Comment:
The actual response seems more like following.

```suggestion
                                    value = "{\n  \"releasesWithSingleCLI\": [\"Release1\", \"Release2\"],\n  \"releasesWithMultipleCLI\": [\"Release3\"],\n  \"STATUS\": \"FAILURE\"\n}"
```